sdk/core/Azure.Core/src/DynamicData/MutableJsonElement.cs

```diff
@@ -469,18 +481,24 @@ public bool TryGetDateTime(out DateTime value)
 
             if (Changes.TryGetChange(_path, _highWaterMark, out MutableJsonChange change))
             {
+                change.EnsureString();
```

Comment:
Does date-time functionality also support unixtime formats i.e., a number (seconds of Unix file time, IIRC)?

Reply:
JsonElement considers DateTime to be a string (see https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.getdatetime?view=net-7.0) -- we built unix date-time capabilities into DynamicData, though, for those who want them.  See `dateTimeFormat` param: https://learn.microsoft.com/en-us/dotnet/api/azure.azurecoreextensions.todynamicfromjson?view=azure-dotnet